src/plugins/usage_collection/server/routes/stats/stats.ts

```diff
@@ -103,7 +103,7 @@ export function registerStatsRoute({
         if (shouldGetUsage) {
           const collectorsReady = await collectorSet.areAllCollectorsReady();
           if (!collectorsReady) {
-            return res.customError({ statusCode: 503, body: { message: STATS_NOT_READY_MESSAGE } });
+            // return res.customError({ statusCode: 503, body: { message: STATS_NOT_READY_MESSAGE } });
```

Comment:
Looks like this is the reason core was pinged for a codeowners review -- I'm assuming this was unintentional?

Reply:
Yes, apologies!

Reply:
all good 🙂